distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java

```diff
@@ -446,21 +488,77 @@ private boolean hasExpectedMetacardsReturned(String queryResult,
     }
 
     /**
+     * Validates that the query matched the expected result count.
+     *
+     * @param queryResult   - The result obtained from sending the query
+     * @param expectedValue - The values expected within the result
+     * @return true if the {@code numberOfRecordsMatched} matches the expected value
+     * @throws Exception if an error occurs parsing the XML response
+     */
+    private boolean hasExpectedMatchCount(String queryResult, ExpectedResultPair expectedValue)
+            throws Exception {
+
+        String originalCount = XmlSearch.evaluate("//@numberOfRecordsMatched", queryResult);
+
+        return originalCount.equals(expectedValue.value);
+    }
+
+    /**
      * Validates that the query returned the expected result count.
      *
      * @param queryResult   - The result obtained from sending the query
      * @param expectedValue - The values expected within the result
-     * @return
-     * @throws Exception
+     * @return true if the {@code numberOfRecordsReturned} matches the expected value
+     * @throws Exception if an error occurs parsing the XML response
      */
     private boolean hasExpectedResultCount(String queryResult, ExpectedResultPair expectedValue)
             throws Exception {
 
-        String originalCount = XmlSearch.evaluate("//@numberOfRecordsMatched", queryResult);
+        String originalCount = XmlSearch.evaluate("//@numberOfRecordsReturned", queryResult);
 
         return originalCount.equals(expectedValue.value);
     }
 
+    private String getPagingMaxRecordsQuery(int maxRecords) {
+        String rawCswQuery = savedCswQueries.get("CswPagingTestLikeQuery");
+        StrSubstitutor strSubstitutor = new StrSubstitutor(ImmutableMap.of("maxRecords",
+                "" + maxRecords));
+
+        strSubstitutor.setVariablePrefix(RESOURCE_VARIABLE_DELIMETER);
+        strSubstitutor.setVariableSuffix(RESOURCE_VARIABLE_DELIMETER);
+        return strSubstitutor.replace(rawCswQuery);
+    }
+
+    private Set<String> ingestPagingRecords(int number) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
❓ Any reason we're building the CSW record inline instead of using a resource file and [`getFileContent()`](https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L694)? I thought we were trying to avoid manually building XML documents in itests.